client/app/cart/cart.controller.js

```diff
@@ -1,26 +1,35 @@
 'use strict';
 
 angular.module('stackStoreApp')
-  .controller('CartCtrl', function ($scope) {
+  .controller('CartCtrl', function ($scope, Cart, Auth) {
+
+    var cartCtrlScope = $scope;
+
     $scope.message = 'Hello';
     $scope.cartTotal;
+    $scope.cart;
+    $scope.populatedCart;
 
-    $scope.cart = [
-    {name: 'Time Machine', price: 100000, description: {blurb: "A time machine", full:"More than the last"}, qty: 1 },
-    {name: 'Watch', price: 10, description: {blurb: "A time machine", full:"More than the last"}, qty: 1},
-    {name: 'Clock', price: 5, description: {blurb: "A time machine", full:"More than the last"}, qty: 1},
-    ];
-
-    $scope.calculateTotal = function(){
+// Might delete this function (trying to work from cart factory)
```

Comment:
yeah this could be done through something like the virtual chuck wrote for Order